sycl/test-e2e/Graph/NativeCommand/invalid.cpp

```diff
@@ -2,10 +2,7 @@
 // RUN: %{run} %t.out
 // RUN: %if preview-breaking-changes-supported %{ %{build} -fpreview-breaking-changes -o %t2.out %}
 // RUN: %if preview-breaking-changes-supported %{ %{run} %t2.out %}
-// REQUIRES: cuda
-
-// XFAIL: preview-mode
-// XFAIL-TRACKER: https://github.com/intel/llvm/issues/18911
+// REQUIRES: target-nvidia, cuda_dev_kit
```

Comment:
Do we really need `cuda_dev_kit` feature for this test? I don't see CUDA SDK headers/library being used in this test.

Also, just out of curiosity, what's the difference between `REQUIRES: target-nvidia` vs. `REQUIRES: cuda` LIT feature? IIRC, we no longer support targeting NVidia GPUs via HIP/ROCm, so `REQUIRES: target-nvidia` and `REQUIRES: cuda` should be identical?

Reply:
I see. It turns out that `target-nvidia` feature is for build-only mode while `cuda` feature would only affect the test run. So, that's why `REQUIRES: target-nvidia` skips building the test when cuda is not available.